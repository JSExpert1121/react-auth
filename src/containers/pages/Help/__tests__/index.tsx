import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16/build';
import { createShallow } from '@material-ui/core/test-utils';

import HelpPage from '../index';
configure({ adapter: new Adapter() });



describe('containers/pages/Help', () => {
    const shallow = createShallow();

    const wrapper = shallow(<HelpPage />)
    test('snapshot test', () => {
        expect(wrapper).toMatchSnapshot();
    })
})