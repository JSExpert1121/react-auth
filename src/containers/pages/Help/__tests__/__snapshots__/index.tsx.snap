// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/pages/Help snapshot test 1`] = `ShallowWrapper {}`;
